packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -60,7 +80,7 @@ describe('request-queue tests', () => {
 
     chai.assert(callbacks.requestWillEnqueue.calledOnce);
     chai.assert(callbacks.requestWillEnqueue.calledWith(
-        sinon.match.has('request')));
+      sinon.match.has('request')));
```

Comment:
This indentation of 4 spaces is intentional as it's a line continuation:
https://google.github.io/styleguide/jsguide.html#formatting-indent